src/import.c

```diff
@@ -242,25 +246,73 @@ CK_RV import_ecdsa(CK_FUNCTION_LIST  *funcs, CK_SESSION_HANDLE h_session, EVP_PK
         return CKR_BUFFER_TOO_SMALL;
     }
     ptr = ec_point;
+    //ptr += 2;
     ec_point_len = i2o_ECPublicKey(ec, &ptr);
+    //ec_point[0] = 0x04;
+    //ec_point[1] = 0x41;
+    //ec_point_len += 2;
+    fprintf(stdout, "EC point %ld\n", ec_point_len);
 
     bn = EC_KEY_get0_private_key(ec);
     if(BN_num_bytes(bn) > sizeof(ec_value)) {
         fprintf(stdout, "Error: EC value too large\n");
         return CKR_BUFFER_TOO_SMALL;
     }
     ec_value_len = BN_bn2bin(bn, ec_value);
+    fprintf(stdout, "EC value %ld\n", ec_value_len);
```

Comment:
This is a CWE-20 due to use of fprintf() inherent weakness to local side-channel attacks but probably very difficult to exploit. Cleaning this up is left as technical debt for now.